compiler/package.json

```diff
@@ -70,7 +70,6 @@
     "@segment/analytics.js-integration-gtag": "file:../integrations/gtag",
     "@segment/analytics.js-integration-gosquared": "file:../integrations/gosquared",
     "@segment/analytics.js-integration-heap": "file:../integrations/heap",
-    "@segment/analytics.js-integration-hellobar": "file:../integrations/hellobar",
```

Comment:
These deletes seem unrelated to your changes, do we need them in this PR ?

Reply:
Indeed it's cleanup work but I think we should keep them. Otherwise, people will always have to troubleshoot the compiler before being able to start working.